### PR TITLE
Implement replies

### PR DESCRIPTION
**Pull request by tripledoublev**
Working on _fediversed replies_

### CODE REVIEW

**Comment by RangerMauve** [comment]
Sent a reply to "reader is out post" lets delete all the extra template stuff other than replies for now

**Comment by tripledoublev** [comment]
Deleted all the extra template _stuff_ in order to focus on replies

**Comment by tripledoublev** [comment]
image

**Comment by tripledoublev** [comment]
Changed the base branch since we do not have moderation enabled on staging.
We could still merge to `staging` and then to `main` but we won't be able to confirm replies on `staging`.
Wondering if the activity pub implementation on `staging` is still relevant

File: _includes/activity_pub/text_modal.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{%- comment -%}
```
Comment (RangerMauve) [inline _includes/activity_pub/text_modal.html]: Can this be removed?

File: _includes/activity_pub/login_modal.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{%- comment -%}
```
Comment (RangerMauve) [inline _includes/activity_pub/login_modal.html]: Can this be removed?

File: _includes/activity_pub/image_modal.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{%- comment -%}
```
Comment (RangerMauve) [inline _includes/activity_pub/image_modal.html]: Is this being used?

**Review by RangerMauve — CHANGES_REQUESTED** [review]
Mind going over the files and removing any that aren't strictly necessary?

File: _includes/activity_pub/text_modal.html (hunk 1 above)
Comment (tripledoublev) [reply _includes/activity_pub/text_modal.html]: Yes.

File: _includes/activity_pub/login_modal.html (hunk 2 above)
Comment (tripledoublev) [reply _includes/activity_pub/login_modal.html]: Also yes :)

File: _includes/activity_pub/image_modal.html (hunk 3 above)
Comment (tripledoublev) [reply _includes/activity_pub/image_modal.html]: No, I removed it
ea11f2b76a8e4338f34370146e2dcf7984c8bf7f

**Comment by tripledoublev** [comment]
Removed all unnecessary modals
ea11f2b76a8e4338f34370146e2dcf7984c8bf7f

**Comment by tripledoublev** [comment]
Looks like I removed something I should not have removed. Will look into it after our C-lab call

**Pull request merged**